store-implementation/parquet-store/README.md

```diff
@@ -12,6 +12,9 @@ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
 limitations under the License.
 
+# Deprecated
+The Parquet Store has been [deprecated](https://github.com/gchq/Gaffer/issues/2604). It will be deleted during the Gaffer 2 development process and Gaffer v2.0.0 will not have a Parquet Store implementation included with it.  
```

Comment:
Technically it's already been deleted during the development process. Correct to say it will not be in Gaffer 2, as there is not yet a non development release of that.

Reply:
Yeah it's weird, kinda a ret-con but perhaps it would be more correct to say "has been deleted"